src/tabs/interfaces.ts

```diff
@@ -134,6 +138,13 @@ export namespace TabsProps {
      * using the `activeTabHref` property of the `change` event's detail.
      */
     href?: string;
+    /**
+     * Determines whether tab content is rendered (but hidden) when the tab is inactive:
```

Comment:
I would remove the "(but hidden)" or rephrase, because technically the content is not hidden if the tab is active. Can maybe say "(but hidden when the tab is not active)".

Reply:
updated